spec/controllers/concerns/idv/step_indicator_concern_spec.rb

```diff
@@ -11,36 +11,82 @@
   before { stub_sign_in(user) }
 
   describe '#step_indicator_steps' do
+    def force_gpo
+      idv_session = instance_double(Idv::Session)
+      allow(idv_session).to receive(:method_missing).
+        with(:address_verification_mechanism).
+        and_return('gpo')
+      allow(controller).to receive(:idv_session).and_return(idv_session)
+    end
+
     subject(:steps) { controller.step_indicator_steps }
 
-    it 'returns doc auth steps' do
-      expect(steps).to eq Idv::Flows::DocAuthFlow::STEP_INDICATOR_STEPS
-    end
+    context 'without an in-person proofing component' do
+      let(:doc_auth_step_indicator_steps) do
+        [
+          { name: :getting_started },
+          { name: :verify_id },
+          { name: :verify_info },
+          { name: :verify_phone_or_address },
+          { name: :secure_account },
+        ]
+      end
 
-    context 'with pending profile' do
-      let(:profile) { create(:profile, deactivation_reason: :gpo_verification_pending) }
+      let(:doc_auth_step_indicator_steps_gpo) do
+        [
+          { name: :getting_started },
+          { name: :verify_id },
+          { name: :verify_info },
+          { name: :get_a_letter },
+          { name: :secure_account },
+        ]
+      end
 
-      it 'returns doc auth gpo steps' do
-        expect(steps).to eq Idv::Flows::DocAuthFlow::STEP_INDICATOR_STEPS_GPO
+      context 'without a pending profile' do
+        it 'returns doc auth steps' do
+          expect(steps).to eq doc_auth_step_indicator_steps
+        end
       end
-    end
 
-    context 'with gpo address verification method' do
-      before do
-        idv_session = instance_double(Idv::Session)
-        allow(idv_session).to receive(:method_missing).
-          with(:address_verification_mechanism).
-          and_return('gpo')
-        allow(controller).to receive(:idv_session).and_return(idv_session)
+      context 'with a pending profile' do
+        let(:profile) { create(:profile, deactivation_reason: :gpo_verification_pending) }
+
+        it 'returns doc auth gpo steps' do
+          expect(steps).to eq doc_auth_step_indicator_steps_gpo
+        end
       end
 
-      it 'returns doc auth gpo steps' do
-        expect(steps).to eq Idv::Flows::DocAuthFlow::STEP_INDICATOR_STEPS_GPO
+      context 'with gpo address verification method' do
+        before { force_gpo }
+
+        it 'returns doc auth gpo steps' do
+          expect(steps).to eq doc_auth_step_indicator_steps_gpo
+        end
       end
     end
 
     context 'with in person proofing component' do
-      context 'with proofing component via pending profile' do
+      let(:in_person_step_indicator_steps) do
+        [
+          { name: :find_a_post_office },
+          { name: :verify_info },
+          { name: :verify_phone_or_address },
+          { name: :secure_account },
+          { name: :go_to_the_post_office },
+        ]
+      end
```

Comment:
Since we're in the process of making changes to various flows and eventually removing them altogether, I wonder if  adding a check for duplicate steps in a flow would be more useful and less maintenance work in the future than hard-coding flows in tests.

Reply:
Per slack conversation with @soniaconnolly, declining this as part of this ticket, but I agree it's a good idea. I will open a ticket for it, to be prioritized and eventually tackled in the normal course of work.